lib/logr.rb

```diff
@@ -0,0 +1,45 @@
+require "logger"
+
+require "logr/version"
+require "logr/entry"
+require "logr/json_formatter"
+
+module Logr
+  class Logger
+    def initialize(name, level=::Logger::DEBUG, log_device=STDOUT)
+      @logger = ::Logger.new(log_device).tap do |logger|
```

Comment:
Yes it was intentional, I don't like `logdev` too much. But we can keep it for historical reasons, I'll survive. :wink: